### PR TITLE
docs: update the README to include correct env variables

### PR DESCRIPTION
**Pull request by harskum**

### CODE REVIEW

**Review by Mohammer5 — CHANGES_REQUESTED** [review]
Hello @harskum! Thank you for the PR :heart: 

I have left these values empty intentionally as these are different from instance to instance.
The values you've selected require a very specific setup (local instance & admin:district user). While these values might make sense when using a local instance, my preference is to keep it instance/user agnostic.

You could add a link to the documentation that describes how to add the correct values though: https://cli-utils-cypress.dhis2.nu/#/guides/add-login-credentials (the docs contain the values you've used as well), I think it'd be a nice addition to the README.

**Comment by harskum** [comment]
@Mohammer5 I have made the changes you requested. Let me know if there are any more concerns around this PR :)

File: README.md [hunk 1]
```diff
@@ -10,16 +10,16 @@ You will need a `cypress.env.json` file with the contents:
...
+    "dhis2Username": "",
+    "dhis2Password": "",
+    "dhis2DataTestPrefix": "dhis2-smsconfiguration"
 }
 ```
 
-The value for the first three fields needs to be appropriate for the
-server you wish to test against. The `dhis2_datatest_prefix` needs to
-be correct for this app.
+The value for the first three fields needs to be appropriate for the server you wish to test against. Check out [this](https://cli-utils-cypress.dhis2.nu/#/guides/add-login-credentials) documentation for more details
```
Comment (Mohammer5) [inline README.md:20]:
```suggestion
The values for the first three fields need to be appropriate for the server you wish to test against. Check out [this](https://cli-utils-cypress.dhis2.nu/#/guides/add-login-credentials) documentation for more details
```

**Review by Mohammer5 — CHANGES_REQUESTED** [review]
That looks good! Just a small typo, then we can merge it

**Review by Mohammer5 — APPROVED** [review]

**Comment by Mohammer5** [comment]
Merged, thank you @harskum!

**Comment by harskum** [comment]
Thanks @Mohammer5!!! I was wondering if you could take a look at this issue? - https://github.com/dhis2/sms-configuration-app/issues/290

It's completely okay if not! I'll try to debug more on this in that case

**Pull request merged**